semver.md

```diff
@@ -1,4 +1,4 @@
-Semantic Versioning 2.0.0
+Semantic Versioning 2.1.0
```

Comment:
I think you could argue this means 3.0.0. I dunno. Something to think about. Semver has never really been super consistent with versioning itself. Maybe it's a good time to start that.

Reply:
yeah, i figured we'd iterate on this line :-) you could argue this is technically *loosening* constraints on v0, or increasing them. Making it v3 would certainly be a safer approach.